operators/src/processing/meteosat/mod.rs

```diff
@@ -101,27 +101,19 @@ mod test_util {
         let mut props = RasterProperties::default();
 
         if let Some(v) = channel {
-            props
-                .properties_map
-                .insert(new_channel_key(), RasterPropertiesEntry::Number(v.as_()));
+            props.insert_property(new_channel_key(), RasterPropertiesEntry::Number(v.as_()));
         }
 
         if let Some(v) = satellite {
-            props
-                .properties_map
-                .insert(new_satellite_key(), RasterPropertiesEntry::Number(v.as_()));
+            props.insert_property(new_satellite_key(), RasterPropertiesEntry::Number(v.as_()));
         }
 
         if let Some(v) = slope {
-            props
-                .properties_map
-                .insert(new_slope_key(), RasterPropertiesEntry::Number(v));
+            props.insert_property(new_slope_key(), RasterPropertiesEntry::Number(v));
```

Comment:
shouldn't this set the `props.scale`?

Reply:
well thats an interesting question. You argued that we should separate the "real" GDAL 'offset' and 'scale' values from values that might be stored as properties. Lat time i checked the MSG driver exposes the "calibration_offset" and "calibration_slope" values as properties and not via the dedicated 'offset' 'scale' values. So i think we should keep it like this since this is what will be used with MSG data.